test/basic.js

```diff
@@ -96,3 +96,29 @@ test('mirror into a readonly drive', async function (t) {
     t.is(error.message, 'Destination must be writable')
   }
 })
+
+test('mirror a drive but file got quickly deleted', async function (t) {
+  const { local, hyper } = await createDrives(t)
+
+  const actual = []
+  const expected = await changeDrive(local)
+
+  const m = new MirrorDrive(local, hyper, { includeEquals: true })
+
+  let first = true
+
+  for await (const diff of m) {
+    if (first && diff.op === 'change') {
+      first = false
+      await local.del(diff.key)
+    }
+
+    delete diff.count
+    actual.push(diff)
+  }
+
+  t.alike(m.count, { files: 6, add: 1, remove: 1, change: 3 })
```

Comment:
Currently, when the file stopped existing it still counts as a change, I think it's ok because the yield was also done, so it's consistent in that way

At the same time sounds wrong because there was no change at all, etc just a yield